lib/optimove.dart

```diff
@@ -0,0 +1,255 @@
+import 'dart:async';
```

Comment:
Should we add docblocks to the public API methods?

Reply:
I suggest to tackle docs in the next version

Reply:
I think docblocks affect the 'score' on the pub.dev site, not but much I think, but is it important?